bibernate-examples/src/main/java/com/bobocode/petros/bibernate/persondemo/Product.java

```diff
@@ -0,0 +1,14 @@
+package com.bobocode.petros.bibernate.persondemo;
+
+import com.bobocode.petros.bibernate.annotations.Entity;
+import com.bobocode.petros.bibernate.annotations.Id;
+import com.bobocode.petros.bibernate.annotations.Table;
+
+@Entity
+@Table(name = "products")
+public class Product {
```

Comment:
Looks like you've started writing an example but its not finished. I would suggest keeping different entities in examples module not for free, but prepare some tests/examples using it. Otherwise it can be removed for now.

Reply:
Done